generate/sre-authorization/blacklist.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "apiGroup": "rbac.authorization.k8s.io",
+        "resources": ["clusterrolebindings","rolebindings"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "machineconfiguration.openshift.io",
+        "resources": ["machineconfigs","machineconfigpools","controllerconfigs"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "machine.openshift.io",
+        "resources": ["machines","machinesets"],
```

Comment:
should we add `machineclass` as well?

Reply:
We could, didn't see any resources in there so didn't put it in.